CHANGELOG.rst

```diff
@@ -2,6 +2,60 @@
 CHANGELOG
 =========
 
+2.8.0
+=====
+
+**ENHANCEMENTS**
+
+* Print a warning when using NICE DCV on micro or nano instances.
+* Add support for S3 URI in custom_chef_cookbook.
+* Make `pcluster update` command safe.
+* Support DCV in China partition.
+* Use describe_instance_types to validate whether an instance type is EFA-enabled.
+* Enable support for ARM instances on Ubuntu 18.04 and Amazon Linux 2.
+* Support EFS in the China partition.
+* Enable configuration of the automatic backup features of FSx file systems.
+* Install  PMIx v3.1.5 and provide slurm support for it on all supported operating systems except for
+  CentOS 6.
+* Install glibc-static, which is required to support certain options for the Intel MPI compiler.
+
+**CHANGES**
+
+* Migrate S3 URLs to Virtual Hosted Style Access.
+* Remove hardcoded account IDs from util/generate-ami-list.py.
+* Avoid validation for ParallelCluster required additional IAM policies.
+* Removed unused dependency on supervisor from the Batch Dockerfile.
+* Move all LogGroup definitions in the CloudFormation templates into the CloudWatch substack.
+* Drop dependency on the gems ridley and ffi-libarchive.
+* Disable libvirtd service on CentOS 7. Virtual bridge interfaces are incorrectly detected by Open MPI and
+  cause MPI applications to hang, see https://www.open-mpi.org/faq/?category=tcp#tcp-selection for details 
+* Use CINC instead of Chef for provisioning instances. See https://cinc.sh/about/ for details.
+* Retry when mounting an NFS mount fails.
+* Install the `pyenv` virtual environments used by ParallelCluster cookbook and node daemon code under
+  /opt/parallelcluster instead of under /usr/local.
+* Avoid downloading the source for `env2` at installation time.
```

Comment:
Is it really worth to be mentioned? we don't need to report here all the details of -node and -cookbook projects